src/app/workspace/resource/representation/video/video.component.html

```diff
@@ -66,15 +66,13 @@
                     {{ muted ? "volume_mute" : "volume_up" }}
                 </mat-icon>
             </button>
-            <button mat-icon-button (click)="toggleCinemaMode()"
-                [matTooltip]="(cinemaMode ? 'Default view' : 'Cinema mode')" [matTooltipPosition]="matTooltipPos">
-                <mat-icon>{{
-                    cinemaMode ? "fullscreen_exit" : "fullscreen"
-                }}</mat-icon>
-            </button>
             <button mat-icon-button (click)="openReplaceFileDialog()" class="replace-file" matTooltip="Replace video file" [matTooltipPosition]="matTooltipPos">
                 <mat-icon>cloud_upload</mat-icon>
             </button>
+            <button mat-icon-button (click)="toggleCinemaMode()"
+                [matTooltip]="(cinemaMode ? 'Default view' : 'Cinema mode')" [matTooltipPosition]="matTooltipPos">
```

Comment:
Maybe using the same word -> 'Default mode' & 'Cinema mode' OR 'Default view' & 'Cinema view'